tests/api2/test_305_container.py

```diff
@@ -136,36 +143,422 @@ def test_10_get_new_public_image_id(request):
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    job_status = wait_on_job(results.json(), 300)
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+
+
+def test_33_verify_ix_chart_config_livenessProbe(request):
+    depends(request, ['tc_livenessProbe'])
+    results = GET(f'/chart/release/id/{tc_release_id}/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    livenessProbe = results.json()['config']['livenessProbe']
+    assert 'ls /usr/share' in livenessProbe['command'], results.text
+    assert livenessProbe['initialDelaySeconds'] == 10, results.text
+    assert livenessProbe['periodSeconds'] == 15, results.text
+
+
+@pytest.mark.dependency(name='tc_containerCAE')
+def test_34_set_truecommand_ix_chart_tc_container_Command_Args_EnvironmentVariables(request):
+    depends(request, ['tc_chart_release'])
+    payload = {
+        'values': {
+            'containerCommand': ['sleep'],
+            'containerArgs': ['infinity'],
+            'containerEnvironmentVariables': [
+                {
+                    'name': 'Eric_Var',
+                    'value': 'Something'
+                }
+            ]
+        }
+    }
+    results = PUT(f'/chart/release/id/{tc_release_id}/', payload)
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), int), results.text
+    job_status = wait_on_job(results.json(), 300)
+    assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+
+
+def test_35_verify_ix_chart_config_container_Command_Args_EnvironmentVariables(request):
+    depends(request, ['tc_containerCAE'])
+    results = GET(f'/chart/release/id/{tc_release_id}/')
+    assert results.status_code == 200, results.text
+    assert isinstance(results.json(), dict), results.text
+    config = results.json()['config']
+    assert 'sleep' in config['containerCommand'], results.text
+    assert 'infinity' in config['containerArgs'], results.text
+    assert {'name': 'Eric_Var', 'value': 'Something'} in config['containerEnvironmentVariables'], results.text
+
+
+@pytest.mark.dependency(name='hostpath-mountpath')
+def test_36_create_datasets_hostpath_and_mountpath(request):
+    depends(request, ['pool_04'], scope='session')
+    result = POST('/pool/dataset/', {'name': f'{pool_name}/tc-hostpath'})
+    assert result.status_code == 200, result.text
+    result = POST('/pool/dataset/', {'name': f'{pool_name}/tc-mountpath'})
+    assert result.status_code == 200, result.text
+
+
+@pytest.mark.dependency(name='tc_volumes')
+def test_37_set_truecommand_ix_chart_volumes(request):
+    depends(request, ['tc_chart_release'])
+    global payload
+    payload = {
+        'values': {
+            'volume': [
+                {
+                    'mountPath': f'/mnt/{pool_name}/tc-mountpath',
+                    'datasetName': f'{pool_name}/tc-mountpath',
```

Comment:
Okay so you don't really have to specify `pool_name/tc-mountpath` here, the `datasetName` here means that system will automatically generate a dataset for this specific volume under the chart release volumes dataset, so what will happen is that dataset created would be `pool/ix-applications/releases/releasename/volumes/herecomesthedatasetname` and mount it inside the pod at specified `mountPath`.